calico-enterprise/networking/configuring/node-local-dns-cache.mdx

```diff
@@ -36,8 +31,8 @@ deployed to your cluster
 ## How to
 
```

Comment:
### Create a policy to allow traffic from NodeLocal DNSCache

The following is a sample network policy that allows all incoming TCP traffic....